src/components/confirmation-box/delete-confirmation-box-content.tsx

```diff
@@ -1,67 +1,45 @@
 import { IdentityTypes } from "@fewlines/connect-management";
-import { useRouter } from "next/router";
 import React from "react";
 import styled from "styled-components";
 
 import { Button, ButtonVariant } from "../buttons/buttons";
-import { fetchJson } from "@src/utils/fetch-json";
 import { getIdentityType } from "@src/utils/get-identity-type";
 
 interface DeleteIdentityProps {
   setOpen: React.Dispatch<React.SetStateAction<boolean>>;
   type: IdentityTypes;
   value: string;
+  onPress: () => void;
 }
 
 const DeleteConfirmationBoxContent: React.FC<DeleteIdentityProps> = ({
   setOpen,
   value,
   type,
-}) => {
-  const router = useRouter();
-
-  return (
-    <>
-      <DeleteConfirmationText>
-        You are about to delete {value}
-      </DeleteConfirmationText>
-      <Button
-        type="button"
-        variant={ButtonVariant.DANGER}
-        onClick={async () => {
-          const requestData = {
-            type: getIdentityType(type),
-            value,
-          };
-
-          await fetchJson(
-            "/api/identity/delete-identity",
-            "DELETE",
-            requestData,
-          ).then(() => {
-            router && router.push("/account/logins");
-          });
-        }}
-      >
-        Delete this{" "}
-        {getIdentityType(type) === IdentityTypes.PHONE
-          ? "phone number"
-          : "email address"}
-      </Button>
-      <Button
-        type="button"
-        variant={ButtonVariant.SECONDARY}
-        onClick={() => setOpen(false)}
-      >
-        Keep{" "}
-        {getIdentityType(type) === IdentityTypes.PHONE
-          ? "phone number"
-          : "email address"}
-      </Button>
-    </>
-  );
-};
-
+  onPress,
+}) => (
+  <>
+    <DeleteConfirmationText>
+      You are about to delete {value}
+    </DeleteConfirmationText>
+    <Button type="button" variant={ButtonVariant.DANGER} onClick={onPress}>
```

Comment:
I removed the function from this component because:
-  I think it should handle only the "display" part
- The logic (the call to the API) has been moved to the login-overview component